providers/teams/pages/members/[id].tsx

```diff
@@ -0,0 +1,215 @@
+import React, { ChangeEvent, useEffect, useState } from 'react'
```

Comment:
Could you please prettier-fy the file? there's already a .prettierrc, you use vs-code so just a single `prettier` extension :)

Reply:
Done